controllers/ibmpowervsmachine_controller.go

```diff
@@ -221,6 +222,15 @@ func (r *IBMPowerVSMachineReconciler) reconcileNormal(machineScope *scope.PowerV
 		case infrav1beta1.PowerVSInstanceStateACTIVE:
 			machineScope.SetReady()
 			conditions.MarkTrue(machineScope.IBMPowerVSMachine, infrav1beta1.InstanceReadyCondition)
+		case infrav1beta1.PowerVSInstanceStateERROR:
+			msg := ""
+			if instance.Fault != nil {
+				msg = instance.Fault.Details
+			}
+			machineScope.SetNotReady()
+			conditions.MarkFalse(machineScope.IBMPowerVSMachine, infrav1beta1.InstanceReadyCondition, infrav1beta1.InstanceErroredReason, capiv1beta1.ConditionSeverityError, msg)
+			capibmrecord.Warnf(machineScope.IBMPowerVSMachine, "FailedBuildInstance", "Failed to build the instance - %s", msg)
```

Comment:
@mkumatag  Overall LGTM, just a minor query - won't this event be recorded with every reconcile run {multiple times} (incase of an errored instance)?

Reply:
For failure it is expected to record multiple times(how many times really happened)